backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/view/ViewConverter.java

```diff
@@ -66,6 +65,16 @@ public List<IRepresentationDescription> convert(View view, List<EPackage> visibl
         return result;
     }
 
+    private Optional<IRepresentationDescription> convert(RepresentationDescription representationDescription, AQLInterpreter aqlInterpreter) {
+        // @formatter:off
+        return this.representationDescriptionConverters.stream()
+                .filter(converter -> converter.canConvert(representationDescription))
+                .map(converter -> converter.convert(representationDescription, aqlInterpreter))
+                .findFirst()
+                .orElse(Optional.empty());
```

Comment:
remove the `orElse(Optional.empty())`

Reply:
converter.convert returns an Optional. I will replace by 
                .flatMap(Optional::stream)
                .findFirst();